.gitignore

```diff
@@ -27,3 +27,5 @@ Servo.app
 
 # VSCode
 .vscode
+
+*.iml
```

Comment:
It's a file automatically created by Android Studio (I use it to test on Android).

I'll remove it from here because that change belongs more to https://github.com/servo/servo/pull/15773, which is about to be merged